src/solidity/types/util.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extractValue: function (slotValue, storageBytes, location) {
+    slotValue = slotValue.replace('0x', '')
+    var offset = slotValue.length - 2 * location.offset - 2 * storageBytes
+    if (offset >= 0) {
+      return '0x' + slotValue.substr(offset, 2 * storageBytes)
+    } else if (offset + 2 * storageBytes > 0) {
+      return '0x' + slotValue.substr(0, 2 * storageBytes + offset)
+    } else {
+      return '0x0'
```

Comment:
I think it is better to return a string of size `storageBytes * 2` in all cases.